composables/view-transition.ts

```diff
@@ -1,4 +1,5 @@
 import type { mastodon } from 'masto'
+import type { ViewTransition } from '#app/plugins/view-transitions.client'
```

Comment:
cannot use `#imports`?

Reply:
remove it, not being used :sweat_smile:

Reply:
Oooops :D At least they were used in a previous attempt :P